.github/workflows/pr-vrt-comment.yml

```diff
@@ -0,0 +1,145 @@
+name: VRT CI | Comment on PR
+on:
+  workflow_run:
+    workflows: ['VRT CI']
+    types:
+      - completed
+
+concurrency:
+  # see https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  NX_PARALLEL: 4 # ubuntu-latest = 4-core CPU / 16 GB of RAM | macos-14-xlarge (arm) = 6-core CPU / 14 GB of RAM
+  NX_PREFER_TS_NODE: true
+  NX_VERBOSE_LOGGING: true
+
+permissions:
+  contents: 'read'
+  actions: 'read'
+
+jobs:
+  run_vr_diff:
```

Comment:
this is preparation of generated screenshots on forks and PR id, that will be used in vrt diffing tool once we have new version